tests/cli/test_subcommands.py

```diff
@@ -11,6 +12,12 @@
 from conda.testing import CondaCLIFixture, PathFactoryFixture, TmpEnvFixture
 
 
+@pytest.fixture(autouse=True)
+def urllib3_logger_critical(caplog):
+    "Increasing the level to critical to prevent retries show up in the stderr"
+    caplog.set_level(logging.CRITICAL, logger="urllib3.connectionpool")
```

Comment:
```suggestion
@pytest.fixture(autouse=True)
def urllib3_logger_error(caplog):
    "Increasing the level to error to prevent retries show up in the stderr"
    caplog.set_level(logging.ERROR, logger="urllib3.connectionpool")
```
should suffice